testing/tests/DevExpress.ui.widgets.editors/datebox.tests.js

```diff
@@ -97,7 +96,7 @@ const moduleConfig = {
 const clearInput = (element, keyboard) => {
     while(element.val()) {
         keyboard.press('backspace');
-        keyboard.press('del'); // Temporary for IE (keyboardMock: caret setting does not work in IE now)
```

Comment:
we should implement clear using caret instead of it.

Smth like:
1. get input text length
2. set caret to {start: 0, end: length}
3. press 'backspace'